working/macros/feature-specification.md

```diff
@@ -871,23 +891,32 @@ Each argument in the metadata annotation for the macro application is converted
 to a form that the corresponding constructor on the macro class expects, which
 it specifies through parameter types:
 
-*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, or `String`
-    (or the nullable forms of any of those), then the argument expression must
-    be a Boolean, number, null, or string literal. Number literals may be
-    negated. String literals may not contain any interpolation, but may be
-    adjacent strings.
+*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, `String`,
+    `List`, or `Map`, (or the nullable forms of any of those), then the argument
+    expression must be a Boolean, number, null, string, list, or map literal.
+
+    * Number literals may be negated.
+    * String literals may not contain any interpolation, but may be adjacent
+      strings.
+    * List and Map literals may only contain any of the supported parameter
```

Comment:
Am I allowed to pass an argument like `[false, 7]`? This would require being able to use a parameter of type `List<Object>`. I think the restrictions on type arguments for Lists, Maps, (and Sets) are currently unspecified.

Reply:
Interesting point, maybe we should allow `List<Object>` (or any built in type?) for the generic types. But then the values must still conform to the rules?

Reply:
Yeah that's not a bad idea.

Reply:
Only enabling `List<Object>` for now, I think that is simple enough.